larray/core/group.py

```diff
@@ -727,6 +727,18 @@ def __init__(self, key, name=None, axis=None):
         # the same name. See test_array.py:test_...
         self.axis = axis
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
```

Comment:
FWIW, this setter approach makes me uneasy. as I always considered the groups to be immutable objects. You could leave that in and we will see how it goes

Reply:
I can move that to init (where `name` is set).